draft-ietf-quic-transport.md

```diff
@@ -2669,7 +2671,36 @@ the same static key (see {{reset-oracle}}).  A connection ID from a connection
 that is reset by revealing the Stateless Reset Token cannot be reused for new
 connections at nodes that share a static key.
 
-Note that Stateless Reset messages do not have any cryptographic protection.
+Note that Stateless Reset packets do not have any cryptographic protection.
+
+
+#### Looping {#reset-looping}
+
+The design of a Stateless Reset is such that it is indistinguishable from a
+valid packet.  This means that a Stateless Reset might trigger the sending of a
+Stateless Reset in response, which could lead to infinite exchanges.  An
+endpoint MUST use any one of the following measures to limit the sending of
+Stateless Reset:
+
+* An endpoint can remember the number of Stateless Reset packets that it has
+  sent and stop generating new Stateless Reset packets once a limit is reached.
+  Using separate limits for different remote addresses will ensure that
+  Stateless Reset packets can be used for some peers even when other peers have
+  exhausted limits.
```

Comment:
I'm not sure if we discussed this anywhere, but if you're willing to keep some state around for a short period of time, wouldn't the easiest thing be to save the last 16 bytes of the packet (for each remote address), and if they match on two (non-identical) packets, you can be pretty sure that those are Stateless Resets.

Reply:
If they use the technique we describe, the input to the calculation for a stateless reset is the connection ID from the inbound packet - and that is randomized in a stateless reset.  So no winner there.